### PR TITLE
Release 1.6

### PR DESCRIPTION
**Pull request by Jamesclark32**
Includes:
 - #63 
 - #68 
 - #70 
 - #71 
 - #73 
 - #75

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -86,6 +86,13 @@ Create a new team account, invite a member of your team, and allow them to spin
 
 == Changelog ==
 
+= 1.6 (2024-02-06) =
+* Add option to purge current URL.
```
Comment (bradt) [inline readme.txt:90]:
```suggestion
* New: Add "Purge this URL" option to our Cache menu in the WordPress nav bar.
```

File: readme.txt [hunk 2]
```diff
@@ -86,6 +86,13 @@ Create a new team account, invite a member of your team, and allow them to spin
...
+* Increase default timeout limits from 1 to 5 seconds.
```
Comment (bradt) [inline readme.txt:91]:
```suggestion
* Improvement: Increase default cache purge timeout limit from 1 to 5 seconds.
```

File: readme.txt [hunk 3]
```diff
@@ -86,6 +86,13 @@ Create a new team account, invite a member of your team, and allow them to spin
...
+* Allow cache key customization.
+* Fix 'Clear All Cache' page cache functionality when Redis errors.
+* Fix "Deprecated: file_exists()" on PHP 8.1.
```
Comment (bradt) [inline readme.txt:94]:
```suggestion
* Improvement: PHP 8.1 compatibility. /props afragen
```

File: readme.txt [hunk 4]
```diff
@@ -86,6 +86,13 @@ Create a new team account, invite a member of your team, and allow them to spin
 
 == Changelog ==
 
+= 1.6 (2024-02-06) =
+* Add option to purge current URL.
+* Increase default timeout limits from 1 to 5 seconds.
+* Allow cache key customization.
```
Comment (bradt) [inline readme.txt:92]:
```suggestion
* New: Cache key customization. /props quimcastella
```

File: readme.txt [hunk 5]
```diff
@@ -86,6 +86,13 @@ Create a new team account, invite a member of your team, and allow them to spin
...
+* Fix 'Clear All Cache' page cache functionality when Redis errors.
```
Comment (bradt) [inline readme.txt:93]:
```suggestion
* Bug Fix: Page cache not cleared when clearing object cache fails.
```

**Review by bradt — CHANGES_REQUESTED** [review]
I've suggested some changes to the changelog entries. Please also reorder them so that it's new first, followed by improvements, then bug fixes.

File: drop-ins/object-cache.php [hunk 6]
```diff
@@ -195,17 +233,16 @@ function wp_cache_init() {
     global $wp_object_cache;
 
     if ( ! defined( 'WP_REDIS_PREFIX' ) ) {
-		define( 'WP_REDIS_PREFIX', get_cache_key_salt());
+			define( 'WP_REDIS_PREFIX', get_cache_key_salt());
```
Comment (A5hleyRich) [inline drop-ins/object-cache.php:236]:
```suggestion
        define( 'WP_REDIS_PREFIX', get_cache_key_salt());
```

File: drop-ins/object-cache.php [hunk 7]
```diff
@@ -195,17 +233,16 @@ function wp_cache_init() {
...
     }
 
     if ( ! defined( 'WP_REDIS_SELECTIVE_FLUSH' ) ) {
-        define( 'WP_REDIS_SELECTIVE_FLUSH', true );
+			define( 'WP_REDIS_SELECTIVE_FLUSH', true );
```
Comment (A5hleyRich) [inline drop-ins/object-cache.php:240]:
```suggestion
        define( 'WP_REDIS_SELECTIVE_FLUSH', true );
```

File: drop-ins/object-cache.php [hunk 8]
```diff
@@ -217,18 +254,18 @@ function wp_cache_init() {
...
+	}
+	if ( getenv( 'SPINUPWP_CACHE_KEY_SALT' ) ) {
+			return getenv( 'SPINUPWP_CACHE_KEY_SALT' );
+	}
+	if ( defined( 'WP_CACHE_KEY_SALT' ) ) {
+			// @deprecated
+			return WP_CACHE_KEY_SALT;
+	}
+
+	return null;
```
Comment (A5hleyRich) [inline drop-ins/object-cache.php:268]:
```suggestion
    if ( defined( 'SPINUPWP_CACHE_KEY_SALT' ) ) {
        return SPINUPWP_CACHE_KEY_SALT;
    }
    if ( getenv( 'SPINUPWP_CACHE_KEY_SALT' ) ) {
        return getenv( 'SPINUPWP_CACHE_KEY_SALT' );
    }
    if ( defined( 'WP_CACHE_KEY_SALT' ) ) {
        // @deprecated
        return WP_CACHE_KEY_SALT;
    }

    return null;
```

**Comment by A5hleyRich** [comment]
Reviewed and tested. Just those few code formatting suggestions.

**Pull request merged**